debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java

```diff
@@ -143,8 +144,17 @@ public Object getValue(PgProto.DatumMessage datumMessage, PgConnectionSupplier c
             case PgOid.FLOAT4:
                 return datumMessage.hasDatumFloat()? datumMessage.getDatumFloat() : null;
             case PgOid.FLOAT8:
-            case PgOid.NUMERIC:
                 return datumMessage.hasDatumDouble() ? datumMessage.getDatumDouble() : null;
+            case PgOid.NUMERIC:
+                if (datumMessage.hasDatumDouble()) {
```

Comment:
Can you add a comment here, stating that this branch is only for supporting older versions of the decoderbufs plug-in and can be removed eventually.

Reply:
Agree